client/packages/invoices/src/Returns/InboundDetailView/Footer/StatusChangeButton.tsx

```diff
@@ -66,13 +63,53 @@ const getStatusOptions = (
   return options;
 };
 
+const getManualStatusOptions = (
+  currentStatus: InvoiceNodeStatus,
+  getButtonLabel: (status: InvoiceNodeStatus) => string
+): SplitButtonOption<InvoiceNodeStatus>[] => {
+  const options: [
+    SplitButtonOption<InvoiceNodeStatus>,
+    SplitButtonOption<InvoiceNodeStatus>,
+    SplitButtonOption<InvoiceNodeStatus>,
+  ] = [
+    {
+      value: InvoiceNodeStatus.New,
+      label: getButtonLabel(InvoiceNodeStatus.New),
+      isDisabled: true,
+    },
+    {
+      value: InvoiceNodeStatus.Delivered,
+      label: getButtonLabel(InvoiceNodeStatus.Delivered),
+      isDisabled: true,
+    },
+    {
+      value: InvoiceNodeStatus.Verified,
+      label: getButtonLabel(InvoiceNodeStatus.Verified),
+      isDisabled: true,
+    },
+  ];
+
+  if (currentStatus === InvoiceNodeStatus.New) {
+    // When the status is new, delivered and verified are available to
+    // select.
+    options[1].isDisabled = false;
```

Comment:
to match the comment, options[2] would have to be undisabled too

Reply:
Oh of course, thank you! 🙏